### PR TITLE
Feature/passenger bookings list

### PR DESCRIPTION
**Pull request by lenchv**

### CODE REVIEW

File: resources/assets/js/app/services/DateService.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    const date = moment(timestamp * 1000),
+        locale = moment().locale(),
+        localeData = moment().locale(locale).localeData(),
+        day = _.padStart(date.date(), 2, '0'),
+        weekday = _.capitalize(localeData.weekdaysShort(date)),
+        month = _.capitalize(localeData.monthsShort(date)),
+        minute = _.padStart(date.minute(), 2, '0'),
+        hour = _.padStart(date.hour(), 2, '0'),
+        now = moment(),
+        time = `${hour}:${minute}`;
```
Comment (TatsuUkraine) [inline resources/assets/js/app/services/DateService.js]: you can use format method from momentjs doc

File: resources/assets/js/app/services/DateService.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import moment from 'moment';
```
Comment (TatsuUkraine) [inline resources/assets/js/app/services/DateService.js]: it's more related to helper than to service

File: resources/assets/js/features/bookings/components/BookingsContainer.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    getData(props, state) {
+        const { filter } = props,
+            {page, limit} = state;
+
+        this.setState({preloader: true});
+
+        BookingService.getBookingsList(filter, page, limit)
+            .then(data => this.setState({
+                data: data.data,
```
Comment (TatsuUkraine) [inline resources/assets/js/features/bookings/components/BookingsContainer.js]: use global state to store data from server, you can set local state based on global one, but main data source in this case should be global

File: resources/assets/js/app/services/BookingService.js [hunk 4]
```diff
@@ -61,6 +61,24 @@ const BookingService = {
             errors.seats = LangService.translate('validate.booking.you_can_book_seats' + LangService.getNumberForm(possibleSeats), {seats: possibleSeats})
         }
         return errors;
+    },
+    getBookingsList(filter = 'upcoming', page = 1, limit = 10) {
+        return securedRequest.get('/api/v1/bookings/' + (filter !== 'upcoming' ? 'past' : 'upcoming' ), {
```
Comment (TatsuUkraine) [inline resources/assets/js/app/services/BookingService.js]: use const for past and upcoming strings

File: resources/assets/js/app/routes.js [hunk 5]
```diff
@@ -76,6 +78,13 @@ export default (store) => {
                     <Route path="edit/:id" component={ Vehicles /*TripEdit*/ }/>
                 </Route>
 
+                {/* Bookings - upcomming and pasts */}
+                <Redirect from='bookings' to='/bookings/upcoming'/>
```
Comment (TatsuUkraine) [inline resources/assets/js/app/routes.js]: you can define upcoming as default filter type, no need to redirect

**Review by TatsuUkraine — CHANGES_REQUESTED** [review]

**Review by sergey-ben — APPROVED** [review]

**Pull request merged**